edexOsgi/com.raytheon.uf.edex.plugin.grid/utility/common_static/base/bundles/grid/CompRefl.xml

```diff
@@ -0,0 +1,45 @@
+<bundle xmlns:ns2="group">
```

Comment:
This file has to do with suggestions from Mike about reflectivity, or something else?

Reply:
Yep, it's the bundle file to load the HRRR Composite Reflectivity with the correct colortable